api/src/main/resources/cb_search_api.yaml

```diff
@@ -447,13 +452,15 @@ definitions:
       hasHierarchy:
         description: Whether criteria is linked to a tree.
         type: boolean
+        default: false
       hasAncestorData:
         description: Whether criteria needs lookup in the criteria_ancestor table.
         type: boolean
+        default: false
       isStandard:
         description: Reveals if this criteria is standard or source.
```

Comment:
@freemabd I think this would make a better enum {standard, source} instead of a boolean.